content/docs/components/ksonnet.md

```diff
@@ -14,8 +14,8 @@ Make sure you have the minimum required version of ksonnet:
 Follow the steps below to install ksonnet:
 
 1. Follow the [ksonnet installation
-   guide][ksonnet-installation], choosing the relevant options for your
-   operating system. For example, if you're on Linux:
+   guide](https://ksonnet.io/get-started/), choosing the relevant options for
+   your operating system. For example, if you're on Linux:
 
     * Set some variables for the ksonnet version:
 
```

Comment:
This may be out of scope for this PR (GitHub literally won't let me comment on the correct line, since it's not part of the diff), but is there a reason you use the `--no-check-certificate` flag on line 31 for the `wget` command that downloads ksonnet? It exposes the user to extra risk of downloading malicious code.

Reply:
I don't know why we use the `--no-check-certificate` flag. That may be worth an issue of its own. It _is_ out of scope for this CL, especially as we use the flag on other pages too.